designer/server/jest.setup.cjs

```diff
@@ -1,2 +1,15 @@
 process.env.SESSION_COOKIE_PASSWORD = 'test-env-session-cookie-password'
 process.env.ROLE_EDITOR_GROUP_ID = 'valid-test-group'
+process.env.MANAGER_URL = 'http://localhost'
+process.env.PREVIEW_URL = 'http://localhost'
+process.env.SESSION_TTL = '86400000'
+process.env.SESSION_COOKIE_TTL = '2419200000'
```

Comment:
Ah I think you've swapped these around by accident?

Do you mind keeping the time code so it's always expressive and human readable?

```suggestion
process.env.SESSION_TTL = Duration.fromObject({ days: 1 })
  .as('milliseconds')
  .toString()
process.env.SESSION_COOKIE_TTL = Duration.fromObject({ minutes: 30 })
  .as('milliseconds')
  .toString()
```

Reply:
🤦  I'm thinking this is a just a key/value properties file, of course it's Javascript! Yes, absolutely we should leave that in.